<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendFrame</class>
 <widget class="QFrame" name="SendFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="m_send_frame_main_layout">
   <item>
    <widget class="QScrollArea" name="m_transfersScrollarea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>846</width>
        <height>373</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="m_send_frame_layout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>445</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Payment ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="m_generatePaymentIdButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Generate Payment ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_paymentIdEdit">
       <property name="maxLength">
        <number>64</number>
       </property>
       <property name="placeholderText">
        <string>Payment identifier issued by payee (exchange, shop or other entity)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="m_advancedWidget">
     <layout class="QGridLayout" name="advancedGridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="1" column="4">
       <widget class="QCheckBox" name="donateCheckBox">
        <property name="text">
         <string>Donate 0.1% to development fund:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Anonymity:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5">
       <widget class="QDoubleSpinBox" name="m_feeSpin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>0.000100000</double>
        </property>
        <property name="maximum">
         <double>99999999.00000000</double>
        </property>
        <property name="singleStep">
         <double>0.00010000</double>
        </property>
        <property name="value">
         <double>0.00010000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="m_priorityGridLayout">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QSlider" name="m_prioritySlider">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>4</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
          <property name="sliderPosition">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="5">
       <widget class="QDoubleSpinBox" name="m_donateSpin">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>99999999.00000000</double>
        </property>
        <property name="singleStep">
         <double>0.10000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="m_mixinLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="m_mixinSlider">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>19</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>7</number>
        </property>
        <property name="sliderPosition">
         <number>7</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Priority:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="m_manualFeeCheckBox">
        <property name="text">
         <string>Manual fee override:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QCheckBox" name="dontRelayCheckBox">
        <property name="text">
         <string>Prepare transaction but do not relay</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_advancedButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Advanced</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_sendAllButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Send All</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/sweep</normaloff>:/icons/sweep</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_clearAllButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Clear All</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_addRecipientButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Add Recipient</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/add_recepient</normaloff>:/icons/add_recepient</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_remote_label">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Wallet is connected through remote node. Additional fee may be applied.</string>
       </property>
       <property name="text">
        <string>Remote mode</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_sendButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/btn-send</normaloff>:/icons/btn-send</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_transfersScrollarea</tabstop>
  <tabstop>m_paymentIdEdit</tabstop>
  <tabstop>m_generatePaymentIdButton</tabstop>
  <tabstop>m_prioritySlider</tabstop>
  <tabstop>m_manualFeeCheckBox</tabstop>
  <tabstop>m_feeSpin</tabstop>
  <tabstop>m_mixinSlider</tabstop>
  <tabstop>donateCheckBox</tabstop>
  <tabstop>m_donateSpin</tabstop>
  <tabstop>m_sendAllButton</tabstop>
  <tabstop>m_clearAllButton</tabstop>
  <tabstop>m_addRecipientButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_addRecipientButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>addRecipientClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_clearAllButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>clearAllClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_sendButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>sendClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_generatePaymentIdButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>generatePaymentIdClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_advancedButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>SendFrame</receiver>
   <slot>advancedClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_prioritySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SendFrame</receiver>
   <slot>priorityValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_mixinSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SendFrame</receiver>
   <slot>mixinValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_feeSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SendFrame</receiver>
   <slot>feeValueChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_manualFeeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_feeSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>780</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_sendAllButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>sendAllClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dontRelayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SendFrame</receiver>
   <slot>dontRelayToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>donateCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SendFrame</receiver>
   <slot>donateToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_manualFeeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SendFrame</receiver>
   <slot>feeOverrideToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_donateSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SendFrame</receiver>
   <slot>donateValueChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendClicked()</slot>
  <slot>sendAllClicked()</slot>
  <slot>clearAllClicked()</slot>
  <slot>addRecipientClicked()</slot>
  <slot>mixinValueChanged(int)</slot>
  <slot>priorityValueChanged(int)</slot>
  <slot>feeValueChanged(double)</slot>
  <slot>donateValueChanged(double)</slot>
  <slot>advancedClicked(bool)</slot>
  <slot>dontRelayToggled(bool)</slot>
  <slot>donateToggled(bool)</slot>
  <slot>feeOverrideToggled(bool)</slot>
 </slots>
</ui>
